src/redux/reducers/forecast/index.js

```diff
@@ -1,14 +1,15 @@
-import {handleActions} from "redux-actions";
-import { forecastActions } from "../../actions/index";
+import { handleActions } from 'redux-actions';
+
+import { forecastActions } from '../../actions';
 
 export default handleActions({
-  [forecastActions.forecastRequest](state) {
+  [forecastActions.forecastRequest] (state) {
     return { ...state, forecastStatus: 'requesting' };
```

Comment:
No need for extra hardcoded values: forecastStatus: 'requesting' . isPending: true/false